app/script/assets/AssetRemoteData.js

```diff
@@ -95,7 +97,12 @@ z.assets.AssetRemoteData = class AssetRemoteData {
   load() {
     let mime_type;
 
-    return this._load_buffer()
+    const loaded_buffer = this._load_buffer();
+    if (!loaded_buffer) {
+      return Promise.reject(new Error());
```

Comment:
It can now: https://github.com/wireapp/wire-webapp/pull/1601/files/7f3b4e3f12da4fc13163dfdf42f02c5b7f11021b#diff-0f4a1910060f26139779c97f4445a4a5R136